### DISCUSSION TITLE
MT3DMS model load

### DISCUSSION THREAD

**Discussion by ArtCoff** [modflowpy/flopy#2244] [Q&A] [answered]
As with loading a water flow model using the load method, is there a way to implement loading an MT3DMS model?

**Comment by mwtoews** [comment, marked as answer]
Did you try [`Mt3dms.load`](https://flopy.readthedocs.io/en/latest/source/flopy.mt3d.mt.html#flopy.mt3d.mt.Mt3dms.load)?
```python
import flopy
mt = flopy.mt3d.mt.Mt3dms.load('example.nam')
```

**Reply by ArtCoff** [reply]
I see，thank you very much.
